### PR TITLE
MOTECH-2859 Can't read field bubble in Key Value Map

### PR DESCRIPTION
**Pull request by piopawel**
Previous solution that was scrolling the bubble was changed to showing an ellipsis and showing a popover.

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by piopawel** [comment]
update branch

**Comment by sebbrudzinski** [comment]
test this please

**Comment by ngraczewski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

File: modules/tasks/tasks/src/main/resources/webapp/css/tasks.css [hunk 1]
```diff
@@ -258,11 +258,34 @@ manipulation-sorter .manipulation .remove:before {
...
+}
+
+.tasks .map-field .popover{
+    color: #000000;
+    visibility: visible;
+}
+
+
+
+
```
Comment (sebbrudzinski) [inline modules/tasks/tasks/src/main/resources/webapp/css/tasks.css]: What's up with all those blank lines?

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**